config/settings/test.py

```
@@ -0,0 +1,25 @@
+"""Test settings for Apatye project."""
+from .base import *  # noqa
+
+DATABASES = {
+    'default': env.db('TEST_DATABASE_URL', default='sqlite:///test.sqlite3'),
```

Ruff (0.14.0)

`env` may be undefined, or defined from star imports

(F405)